test/acceptance/formatters/kakoune-test.js

```diff
@@ -0,0 +1,217 @@
+import { setupProject, teardownProject, runBin } from '../../helpers/bin-tester.js';
+import { getOutputFileContents } from '../../helpers/index.js';
+
+describe('Kakoune formatter', () => {
+  let project;
+  beforeEach(async function () {
+    project = await setupProject();
+    await project.chdir();
+  });
+
+  afterEach(function () {
+    teardownProject();
+  });
+
+  it('should format errors', async function () {
+    await project.setConfig({
+      rules: {
+        'no-bare-strings': true,
+      },
+    });
+    await project.write({
+      app: {
+        templates: {
+          'application.hbs': '<h2>Here too!!</h2> <div>Bare strings are bad...</div>',
+          components: {
+            'foo.hbs': '{{fooData}}',
+          },
+        },
+      },
+    });
+
+    let result = await runBin('.', '--format', 'kakoune');
+
+    expect(result.exitCode).toEqual(1);
+    expect(result.stdout).toMatchInlineSnapshot(`
+      "app/templates/application.hbs:1:4: Error: Non-translated string used (no-bare-strings) 
+      app/templates/application.hbs:1:25: Error: Non-translated string used (no-bare-strings) 
+
+      2 problems"
+    `);
+    expect(result.stderr).toBeFalsy();
+  });
+
+  it('should format errors and warnings', async function () {
+    await project.setConfig({
+      rules: {
+        'no-bare-strings': true,
+        'no-html-comments': 'warn',
+      },
+    });
+    await project.write({
+      app: {
+        templates: {
+          'application.hbs':
+            '<h2>Here too!!</h2><div>Bare strings are bad...</div><!-- bad html comment! -->',
+        },
+      },
+    });
+
+    let result = await runBin('.', '--format', 'kakoune');
+
+    expect(result.exitCode).toEqual(1);
+    expect(result.stdout).toMatchInlineSnapshot(`
+      "app/templates/application.hbs:1:4: Error: Non-translated string used (no-bare-strings) 
+      app/templates/application.hbs:1:24: Error: Non-translated string used (no-bare-strings) 
+      app/templates/application.hbs:1:53: Warning: HTML comment detected (no-html-comments) 
+
+      3 problems"
+    `);
+
+    expect(result.stderr).toBeFalsy();
+  });
+
+  it('should include information about available fixes', async function () {
```

Comment:
I don't see any output about fix information?

Reply:
Kakoune doesn't support fix information so i removed the test.